channeldb/invoices.go

```diff
@@ -1279,6 +1279,14 @@ type InvoiceQuery struct {
 	// Reversed, if set, indicates that the invoices returned should start
 	// from the IndexOffset and go backwards.
 	Reversed bool
+
+	// MinCreationDate, if set, filters out all invoices with a creation
+	// date less than it.
+	MinCreationDate uint64
```

Comment:
I'd suggest using the `time.Time` type here over int64, so that you can use some of the build in functionality like `.IsZero()`

Reply:
Good call! Done.